src/web/src/styles/telescope-post-content.css

```diff
@@ -22,6 +28,33 @@
   padding-bottom: 1rem;
   display: block;
   height: auto;
+  cursor: zoom-in;
+}
+
+.zoomed-image-container {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: 1200;
```

Comment:
Just curious, I remember you use 9999, why 1200 now?

Reply:
9999 was just a random number but all I needed was something bigger than 1100 (the z-index for the navbar)